cookbooks/aws-parallelcluster-environment/attributes/environment.rb

```diff
@@ -55,6 +55,7 @@
 default['cluster']['shared_dir_compute'] = node['cluster']['shared_dir']
```

Comment:
See comments in https://github.com/aws/aws-parallelcluster-cookbook/pull/2678/files